apps/coscrad-frontend/src/app/app.tsx

```diff
@@ -25,6 +27,19 @@ export function App() {
             fetchFreshState(dispatch, aggregateTypeFromMessage);
     };
 
+    const { siteTitle, siteFavicon } = useContext(ConfigurableContentContext);
+
+    useEffect(() => {
+        document.title = siteTitle;
+    }, [siteTitle]);
+
+    const link: HTMLLinkElement =
```

Comment:
Interesting, this doesn't seem very reacty but otherwise I guess you have to use something like Helmet